### DISCUSSION TITLE
<code><br>- First Episode</code>

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#14] [Comments]
# <code><br>- First Episode</code>

My first episode is out!

https://www.raymondcamden.com/2024/02/06/codebr-first-episode

**Comment by fmdano** [comment]
Hey Ray,
Thanks for the beginning of this video...if you are going to do a show like this, personally I think it is a great idea to tell viewers about you and why you code and where you came from...Plus saying you could not do the math and ended up in technical writing and then ended up being the developer you are, that can certainly resonate with many of us...I know I have heard a lot about you in the past, but I learned something new about your education and that to me was great....I hope this show goes well for you and you get out of it what you want because I know your followers will get something out of it, we always do...good luck with this new fun adventure :)

**Reply by cfjedimaster** [reply]
Glad you liked it! :)

**Reply by fmdano** [reply]
Just finished the video...even if I am mostly a CFML developer (yes there are still some of us that just do CFML), it was cool and always cool to see what you are doing and what tools you are using to do the things you do....I'm sure for me and my CFML work I won't have use for a lot of your hands on work,  however, you present great IMO so I will check into them because I might see a way to do something in my work I did not think of as a coder, or get inspired to check into some other language...So, can't wait to see what you come up with in Episode 2 for everyone...I hope this got some good traction for the live sessions....keep up the awesome work :)

**Comment by cebnz** [comment]
Phew. That was exhausting.. but interesting.

We used to say that if you only have a hammer you treat everything like a nail. But what happens when you have an infintely-expanding toolbox?

(I couldn't hack the maths either!)

**Reply by cfjedimaster** [reply]
Heh, sorry, I got on a run and just kept sharing tool after tool. ;)

**Comment by cebnz** [comment]
Ha... no apology needed... it was thought-provoking. Keep it up!

Ian
